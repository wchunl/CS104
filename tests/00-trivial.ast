ROOT "" (0.0.0)
|   TOK_FUNCTION "" (4.3.1) function
|   |   TOK_TYPE_ID "" (4.3.1) typeid
|   |   |   TOK_INT "int" (4.3.1) int
|   |   |   TOK_IDENT "main" (4.3.5) variable
|   |   TOK_PARAM "(" (4.3.9) param
|   |   TOK_BLOCK "{" (4.3.11)
