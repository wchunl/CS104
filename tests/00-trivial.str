string_set[   0]:      21606623943142546 0xb77a78->"{"
string_set[   1]:   13686721134187162920 0xb77b18->"}"
string_set[   3]:   15816506932648033945 0xb77908->"("
                   12597418120160561823 0xb73218->"int"
string_set[   6]:    6142509188972423790 0xb735e8->""
string_set[   7]:   12868445110721718657 0xb779a8->")"
                   11868120863286193964 0xb73608->"main"
load_factor = 0.636
bucket_count = 11
max_bucket_size = 2
