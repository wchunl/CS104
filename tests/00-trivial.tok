#  1 "00-trivial.oc"
#  1 "<built-in>"
#  1 "<command-line>"
#  1 "00-trivial.oc"
   4 3.00001 262 TOK_INT       int
   4 3.00005 282 TOK_IDENT     main
   4 3.00009  40 '('           (
   4 3.00010  41 ')'           )
   4 3.00011 123 '{'           {
   4 3.00012 125 '}'           }
