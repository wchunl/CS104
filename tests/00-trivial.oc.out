; 0.  1: # 1 "00-trivial.oc"; 1.  0: 
; 1.  1: # 1 "<built-in>"
; 2.  1: # 1 "<command-line>"
; 3.  1: # 1 "00-trivial.oc"
; 4.  1: 
; 4.  2: 
; 4.  3: int main(){}
hello